packages/components-react/link-react/package.json

```diff
@@ -32,6 +32,7 @@
     "build": "rollup --config ./rollup.config.mjs",
     "clean": "rimraf ./dist/ ./*.tsbuildinfo ./.rollup.cache/",
     "test": "jest",
+    "test:watch": "jest --watch",
```

Comment:
Is het niet handiger om `watch:test` te doen, zodat je `npm-run-all test:**` kan doen als er meerdere tests zijn?

Reply:
Is denk ik niet de 'conventie'? Als al deze test PRs gemerged zijn heb ik nog eentje met coverage erin… laten we even kijken hoe dat eruit ziet en of er überhaupt een apart test script voor nodig is.